### DISCUSSION TITLE
Exclude folders for nuxi build & nuxi dev

### DISCUSSION THREAD

**Discussion by jcalfee** [nuxt/nuxt#19979] [Questions]
Where can I exclude foldres for `nuxi dev` and `nuxi build`?  I'm trying to use the nuxi 2 to 3 bridge.  It is trying to build code I have in folders covered by `.gitignore` ..

**Comment by TheAlexLichter** [comment]
`.gitignore` is only for git as the name says.
You can use a [`.nuxtignore` file](https://nuxt.com/docs/guide/directory-structure/nuxtignore#nuxt-ignore-file) or the `ignore` option in `nuxt.config.ts`
